xl-v1-bucket.go

```diff
@@ -238,7 +239,14 @@ func (xl xlObjects) DeleteBucket(bucket string) error {
 		// Delete volume inside a go-routine.
 		go func(index int, disk StorageAPI) {
 			defer wg.Done()
-			err := disk.DeleteVol(bucket)
+			// Cleanup all the previously incomplete multiparts.
```

Comment:
better to cleanup incomplete uploads after DeleteVol succeeds. i.e if DeleteVol fails for some reason (ex if it has objects in it) we should not delete the multipart entries.